markdown/developers/sync.md

```diff
@@ -3,26 +3,188 @@ title: Synchronisation
 
-## How it works
-In brief, when the synchronisation tool is triggered it compares every pipeline against the new standards and opens pull requests when mismatches are found.
-These pull requests needs to be manually dealt with by the pipeline maintainers.
+1. [Merging automated PRs](#merging-automated-prs)
+2. [Manual synchronisation](#manual-synchronisation)
+3. [Setting up a pipeline for syncing retrospectively](#setting-up-a-pipeline-for-syncing-retrospectively)
 
-Behind the scenes, the synchronisation tool fetches the variables needed for a pipeline and uses this to trigger a `nf-core create --no-git` command.
-The result from this command is then compared against what is stored in a special branch of the pipeline called the `TEMPLATE` branch.
-If any changes are detected, a pull request is created.
+### How template synchronisation works
+
+The `nf-core` helper tools have a subcommand for synchronising a pipeline with the nf-core template.
+Although this can be run manually, it is usually only used by the GitHub Actions automation:
+when a new version of nf-core/tools is released it runs for all nf-core pipelines and automatically opens pull-requests (PRs) with the necessary changes to every pipeline.
+These pull requests then need to be manually dealt with by the pipeline maintainers.
+
+Behind the scenes, this synchronisation is done by using `git`.
+Each repository has a special `TEMPLATE` branch which contains only the "vanilla" code made by the create tool.
+The synchronisation tool fetches the variables needed for a pipeline and uses this to trigger a `nf-core create --no-git` command with the latest version of the template.
+The result from this is then compared against what is stored in the `TEMPLATE` branch and committed.
+When merging from the `TEMPLATE` branch back in to the main `dev` branch of the pipeline, `git` should be clever enough to know what has been changed since the template was first used, and only present the relevant changes.
 
 For this to work in practice, the `TEMPLATE` branch needs to have a shared git history with the `master` branch of the pipeline.
 The `nf-core create` command arranges this by enforcing a first commit to the `master` branch before any development has taken place.
-The next section deals with the case when the pipeline *was not* created by the `nf-core create` command.
+If the pipeline *was not* created by the `nf-core create` command, this has to be set up manually.
+For instructions on this, see [Setting up a pipeline for syncing retrospectively](#setting-up-a-pipeline-for-syncing-retrospectively).
+
+# Merging automated PRs
+
+When a new release of tools is created, each pipeline will get an automated pull-request (PR) opened to merge the changes to the template in to the pipeline.
+
+If there are no merge conflicts on the PR, then that's great!
+If you're are happy with the changes, feel free to just merge it in to the `dev` branch directly.
```

Comment:
```suggestion
If you are happy with the changes, feel free to just merge it into the `dev` branch directly.
```

Reply:
I have noticed myself writing in to instead of into a lot recently, something strange happening in my brain...